src/lib/components/Widget.tsx

```diff
@@ -39,7 +39,6 @@ const WidgetWrapper = styled.div<{ width?: number | string }>`
   font-size: 16px;
   font-smooth: always;
   font-variant: none;
-  height: 348px;
```

Comment:
This should not be removed - it constrains the height in cases where it would otherwise overflow, like with token lists